client/homescreen/activity-panel/index.js

```diff
@@ -39,16 +39,14 @@ export const ActivityPanel = () => {
 		const countLowStockProducts = getLowStockCount( select );
 		const countUnapprovedReviews = getUnapprovedReviews( select );
 		const publishedProductCount = getSetting( 'publishedProductCount', 0 );
-		const taskLists = select( ONBOARDING_STORE_NAME ).getTaskLists();
+		const taskList = select( ONBOARDING_STORE_NAME ).getTaskList( 'setup' );
 
 		return {
 			countLowStockProducts,
 			countUnapprovedReviews,
 			countUnreadOrders,
 			manageStock,
-			isTaskListHidden: Boolean( taskLists.length )
-				? ! taskLists.find( ( list ) => list.id === 'setup' ).isVisible
-				: null,
+			isTaskListVisible: taskList?.isVisible,
```

Comment:
Changing the `isTaskListHidden` prop here, will also require a change in the `getAllPanels` function here: https://github.com/woocommerce/woocommerce-admin/blob/b957d70a1fe223f0ca7290b861d95f6e0b54a890/client/homescreen/activity-panel/panels.js#L18
None of the panels are showing up at the moment.